stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/reflect/Equals.java

```diff
@@ -212,7 +211,7 @@ private boolean isDeepEqual(BStruct lhsStruct, BStruct rhsStruct, BStructType st
      * @param rhsMap Right hand side map.
      * @return Return if maps are deeply equal.
      */
-    private boolean isDeepEqual(BMap lhsMap, BMap rhsMap) {
+    private boolean isEqual(BMap lhsMap, BMap rhsMap) {
```

Comment:
These should be renamed to "isNotEqual", since you're checking inequality here. Fix in all the places.

Reply:
I negated the return value in commit https://github.com/ballerina-lang/ballerina/pull/4680/commits/a50c2917b32621abedcffe8f3ab6e65bea37fe34 .
So now if values are equal, `isEqual(BMap lhsMap, BMap rhsMap)` will return true, else false.